Handle non notebook based runs (#16)
PrefectHQ/prefect-databricks: CHANGELOG.md

### Security

## 0.1.2

Released on TBD

> ### Edited

Reviewer: ```suggestion
### Added
```

> - `jobs_runs_submit_and_wait_for_completion` flow - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16)

Reviewer: ```suggestion
- Logging of job runs tasks status within `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16)
```

> 

Reviewer: Can we also have another section

```
### Fixed

- Executing Python scripts through `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16)
```

Author: Ideal, will add in the AM

### Security

## 0.1.2

Released on TBD

### Added

> - Logging of job runs tasks status within `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16

Reviewer: ```suggestion
- Logging of job runs tasks status within `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16).
```

### Fixed

> - Executing Python scripts through `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16)

Reviewer: ```suggestion
- Executing Python scripts through `jobs_runs_submit_and_wait_for_completion` - [#16](https://github.com/PrefectHQ/prefect-databricks/pull/16).
```
